composer.lock

```diff
@@ -3764,4 +3764,4 @@
     "platform": [],
     "platform-dev": [],
     "plugin-api-version": "2.0.0"
-}
+}
```

Comment:
Its probably composer which does that, but as a convention could we agree that all files should end with a new line (this is checked for the other files during code style check)? The editors may also be in cause. On my systems, it always saves with empty new line.

Reply:
Hmm, true. But we may not control all library/external files. :)

Reply:
:D